test/settypes/bitset.jl

```diff
@@ -0,0 +1,9 @@
+using SparseConnectivityTracer: ×
+using Test
+
+x = BitSet.(1:10)
+y = (x[1] ∪ x[3]) ∪ (x[3] ∪ ((x[5] ∪ x[7]) ∪ x[1]))
+
+@test sort(collect(y)) == [1, 3, 5, 7]
+@test y × y isa Set{Tuple{Int,Int}}
+@test length(collect(y × y)) == 16
```

Comment:
It would be nice to have a single test-set that loops over all set types at the end instead of individual files for each set type.

Reply:
done